01-getting-started/getting-started.Rmd

```diff
@@ -5,6 +5,15 @@ output:
   html_notebook: 
     toc: true
     toc_float: true
+navbar:
```

Comment:
This isn't working. I think its more complicated than what I tried.

Reply:
I think this needs to go in a separate file.